Assets/Tile walking mini game/Scripts/Timer.cs

```diff
@@ -0,0 +1,91 @@
+using System;
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.Events;
+
+public class Timer : MonoBehaviour
+{
+    [SerializeField] private GridManager gridManager;
+
+    [SerializeField] private int timerDuration;
+
+    private int _timeLeft;
+
+    private int timeLeft
```

Comment:
Convention for "properties" with getters/setters like this is usually PascalCase, even if they are private